tests/sentry/post_process_forwarder/test_post_process_forwarder.py

```diff
@@ -58,10 +58,8 @@ def setUp(self) -> None:
         self.events_topic = f"events-{self.consumer_and_topic_suffix}"
         self.commit_log_topic = f"events-commit-{self.consumer_and_topic_suffix}"
         self.override_settings_cm = override_settings(
-            KAFKA_EVENTS=self.events_topic,
-            KAFKA_TRANSACTIONS=self.events_topic,
-            KAFKA_TOPICS={
-                self.events_topic: {"cluster": "default"},
+            KAFKA_TOPIC_TO_CLUSTER={
+                "events": "default",
```

Comment:
where did the transaction one go on line 62?

Reply:
it's not really needed since it's the default, i removed this events one too